modules/cloud_installations.adoc

```diff
@@ -19,8 +19,45 @@ cluster. These details include ...
 With a customized cluster, you can specify more details of the installation, 
 such as ...
 
+When you install a fully-managed {product-title} cluster, you download the
+installer from link:try.openshift.com. This site manages:
+* Rest API for accounts
+* Registry tokens, which are the pull secrets that you use to obtain the required
+components
+* Cluster registration, which associates the cluster identity to your Red Hat
+account to facilitate usage metrics gathering
+
+In {product-title} version 4.0, the installer is a go binary that performs a
+series of file transformations on a set of assets. With the fully-managed option,
+you delegate the infrastructure bootstrapping and provisioning to the installer
+instead of doing it yourself. Because you do not use the installer to upgrade or
+update your cluster, if you do not highly customize your cluster, you run the
+installer only once.
+
+You use three sets of files during installation: an installation configuration
+file, Kubernetes manifests, and Ingition configurations for your machine types.
+
+The installation configuration file is transformed into Kubernetes manifests, and
+then the manifests are wrapped into Ignition configurations. The installer uses
+these Ingition configurations to create the cluster.
+
+The install configuration files are all pruned when you run the installer,
+so be sure to back up all configuration files that you want to use again.
+
 [IMPORTANT]
 ====
 You cannot modify the parameters that you set during installation. You have to
 ...
 ====
+
+////
+There are individual commands to perform the different actions in cluster creation
+if you want to try to make customizations, but you can run openshift-install
+create cluster to get the default cluster done quick.
+
+$ openshift-install --help
+$ openshift-install create install-config
+$ openshift-install create manifests
+$ openshift-install create ignition-configs
+$ openshift-install create cluster
```

Comment:
i'd suggest to always educate the users to run using `--dir=` option otherwise issues might appear in case multiple clusters are deployed from same host controller.

@wking fyi

Reply:
Example working [here][1].

[1]: https://github.com/openshift/installer/blame/v0.8.0/README.md#L53-L55

Reply:
I will open another PR to update the install docs, and I plan to include the `--dir=` option there.